### PR TITLE
Removing magic constant from template param check

### PR DESCRIPTION
**Pull request by SecMeant**
Storing digits in variable, thus removing magic constant from template param check.

### CODE REVIEW

**Comment by tcsullivan** [comment]
This would be a nice improvement; however, it's leaving us with `digits` and `digit_count` in the global scope, which could mess up other code. One solution to that would be to put everything in a namespace, though I haven't done that yet (not sure what to name it). Open to any other solutions if you have them.

**Comment by Neargye** [comment]
@tcsullivan `namespace cits // constexpr_integral_to_string` ?

File: to_string.hpp [hunk 1]
```diff
@@ -9,6 +9,9 @@
 
 #include <type_traits>
 
+constexpr char digits[] = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ";
```
Comment (Neargye) [inline to_string.hpp:14]: `inline constexpr `

File: to_string.hpp [hunk 2]
```diff
@@ -9,6 +9,9 @@
...
+constexpr auto digit_count = sizeof(digits) / sizeof(digits[0]);
```
Comment (Neargye) [inline to_string.hpp:15]: also `inline constexpr`

**Review by Neargye — CHANGES_REQUESTED** [review]

File: to_string.hpp (hunk 1 above)
Comment (m-peko) [reply to_string.hpp:14]: @Neargye why `inline constexpr` when `constexpr` itself is implicitly inline?

Comment (Neargye) [reply to_string.hpp:14]: ​constexpr specifier implies  inline for static data members as well as functions, but not for global variable. 
https://en.cppreference.com/w/cpp/language/inline

**Comment by SecMeant** [comment]
Now, when I think about it, maybe we should use something like

namespace to_string_ { ... }

Since its only purpose is to only hide digits symbols.
@tcsullivan

**Comment by tcsullivan** [comment]
@SecMeant The `constexpr_to_string` namespace is honestly probably enough. I just realized that the final `to_string` declaration is outside of the namespace, so this isn't actually affecting how `to_string` is called (something I was concerned about).

I'm good to merge this PR as it is now, if you are.

**Comment by SecMeant** [comment]
Sure

**Pull request merged**